csharp/ql/test/library-tests/ir/ir/using.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+class UsingStmt 
+{
+    public class MyDisposable : IDisposable
+    {
+        public MyDisposable() { }
+        public void DoSomething() { }
+        public void Dispose() { }
+    }
+
+    static void Main()
```

Comment:
You can, if you want test this as well:

```
   using var o3 = new MyDisposable();
```
Then you can add a comment anywhere in the file (perhaps, at the bottom)
```
// semmle-extractor-options: /langversion:preview
```

Reply:
I've added the `TranslatedUsingDeclStmt` class back and included the test you mentioned.